src/config.ts

```diff
@@ -65,6 +65,12 @@ export class KubeConfig implements SecurityAuthentication {
         new OpenIDConnectAuth(),
     ];
 
+    // Optionally add additional external authenticators, you must do this
+    // before you load a kubeconfig file that references them.
```

Comment:
Do you think it is worth adding a test that intentionally does not do this (if for no other reason than knowing if it accidentally changes)?

Reply:
I don't think this can ever change, because the kubeconfig file references the authenticator as part of the YAML file, so if we don't know about the authenticator because it hasn't been registered when we load the kubeconfig, there's no way that we can link the authenticator requested in the file to the kubeconfig.

So while we could add a test, I'm not sure it would ever break.